fio/fio_run

```diff
@@ -342,6 +342,7 @@ reduce_data()
 			echo "" >> results.csv
 
 			cat  results.csv >> $working_dir/results_fio.csv
+			rm results.csv
```

Comment:
Woudn't it be simpler to `mv` the file?

Reply:
No, as we are constantly adding to $working_dir/results_fio.csv.  If we move it, we will lose the prior data that we want.  The file is a rollup of all the results for the run.